jsk_rviz_plugins/CMakeLists.txt

```diff
@@ -5,7 +5,14 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(jsk_rviz_plugins)
 if("$ENV{ROS_DISTRO}" STRGREATER "indigo")
-  add_compile_options(-std=c++11)
+  if("$ENV{ROS_DISTRO}" STRGREATER "melodic")
+    # How to require c++14: https://stackoverflow.com/a/42842447
+    set(CMAKE_CXX_STANDARD 14)
+    set(CMAKE_CXX_STANDARD_REQUIRED ON)
+    set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
`CMAKE_CXX_STANDARD` is ntroduced in `cmake>=3.1`
please use `add_compile_options(-std=c++14)`